docs/users_guide/landice/test_groups/ismip6_forcing.rst

```diff
@@ -0,0 +1,302 @@
+.. _landice_ismip6_forcing:
+
+ismip6_forcing
+==============
+
+The ``landice/ismip6_forcing`` test group processes (i.e., remaps and renames)
+the atmospheric and ocean forcing data of the Ice Sheet Model Intercomparison for CMIP6
+(ISMIP6) protocol. The processed data is used to force MALI in its simulations
+under a relevant ISMIP6 (either the 2100 or 2300) experimental protocol.
+The test group includes three test cases, ``atmosphere``, ``ocean_basal`` and
+``ocean_thermal``; the ``atmosphere`` test case has two steps:
+``process_smb`` and ``process_smb_racmo``.  The ``ocean_basal`` and the
+``ocean_thermal`` test case each has one step, ``process_basal_melt``, and
+``process_thermal_forcing``, respectively. (For more details on the steps of
+each test case, see :ref:`landice_ismip6_forcing_atmosphere`,
+:ref:`landice_ismip6_forcing_ocean_basal` and
+:ref:`landice_ismip6_forcing_ocean_thermal`.)
+Approximated time for processing a single forcing file
+on Cori (single core) is 2, 7, and 1 minutes for the atmosphere, ocean basal,
+and ocean thermal testcase, respectively.
+
+Before providing the details of necessary source data of the ISMIP6 protocols,
+we provide a summary of instructions of an overall process of this test group:
+To start off, users need to provide a MALI mesh onto which the source data
+will be remapped and renamed. More information about obtaining these meshes will
+be provided as soon as they are publicly available. Then, for a given MALI mesh,
+
+1. run :ref:`landice_ismip6_forcing_ocean_basal` once, independent of the
+model, scenario and end year.
+
+2. run :ref:`landice_ismip6_forcing_ocean_thermal` once with
+``process_obs_data = True`` in the config file (:ref:`landice_ismip6_forcing_config`)
+to process the thermal forcing from the observational climatology (used for
+control runs).
```

Comment:
Make sure to update the documentation here and elsewhere if you choose to make these changes.